test/unit/alignment/pairwise/fixture/alignment_fixture.hpp

```diff
@@ -62,8 +61,10 @@ struct alignment_fixture
     std::string aligned_sequence1;
     std::string aligned_sequence2;
 
-    seqan3::alignment_coordinate begin_positions;
-    seqan3::alignment_coordinate end_positions;
+    size_t begin1_position;
+    size_t begin2_position;
+    size_t end1_position;
+    size_t end2_position;
```

Comment:
Maybe give the variables clearer names:
```suggestion
    size_t begin_column_position;
    size_t begin_row_position;
    size_t end_column_position;
    size_t end_row_position;
```